randovania/data/json_data/prime3.txt

```diff
@@ -3370,6 +4183,7 @@ Asset id: 17766988414710659705
       Hyper Missile
```

Comment:
Add Missile > 4 just for accuracy

Reply:
IIRC there are bats that fly around that can give drops. I'll add Missile >=1 though - let me know if you agree/disagree, then I'll resolve this appropriately.

Reply:
true there are bats. 4 is just the safer option. though under normal circumstances (assuming regular missile counts, not random starting room, or warping to room, 1 is fine